pulsar-client-cpp/wireshark/pulsarDissector.cc

```diff
@@ -203,7 +277,7 @@ static void dissect_message_metadata(proto_tree* frame_tree, tvbuff_t *tvb, int
     }
 
     static MessageMetadata msgMetadata;
-    uint8_t* ptr = (uint8_t*) tvb_get_ptr(tvb, offset, metadataSize);
+    auto* ptr = (uint8_t*) tvb_get_ptr(tvb, offset, metadataSize);
```

Comment:
Just use `auto` instead of `auto*` if you don't want to declare type explicitly (because there is a conversion in right hand), the type deduction will work well.

Reply:
Addressed.